recipe/conda_build_config.yaml

```diff
@@ -1,4 +1,7 @@
 mpi:
   - nompi
   - mpich  # [not win]
-  - openmpi  # [not win]
\ No newline at end of file
+  - openmpi  # [not win]
+
+c_stdlib_version:
+  - 2.17  # [linux]
```

Comment:
Needed to avoid issue in https://github.com/conda-forge/openmpi-feedstock/issues/143